tests/handlers/test_register.py

```diff
@@ -587,17 +587,16 @@ def test_register_not_support_user(self) -> None:
         self.assertFalse(self.get_success(d))
 
     def test_invalid_user_id(self) -> None:
-        invalid_user_id = "+abcd"
+        invalid_user_id = "^abcd"
         self.get_failure(
             self.handler.register_user(localpart=invalid_user_id), SynapseError
         )
 
-    @override_config({"experimental_features": {"msc4009_e164_mxids": True}})
-    def text_extended_user_ids(self) -> None:
```

Comment:
Was this originally a typo?

Reply:
Yes, the test wasn't even running. 🤦 Hence why I had to change the assertion...